docker/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM alpine:3.10
```

Comment:
All we need to do is update this to `docker.mirror.hashicorp.services/alpine:3.10` to avoid the dockerhub rate limits.

Reply:
Updated, thanks!